ipa-core/src/protocol/prss/mod.rs

```diff
@@ -569,7 +638,7 @@ pub mod test {
     #[test]
     #[cfg(debug_assertions)]
     #[should_panic(
-        expected = "Generated randomness for index '100' twice using the same key 'protocol/test'"
+        expected = "Generated randomness for index '6553600' twice using the same key 'protocol/test'"
```

Comment:
any sense in implementing Display for PrssIndexU128 to make this less of a pain to use?